pkg/minikube/cluster/cluster.go

```diff
@@ -154,6 +156,24 @@ func StartCluster(h sshAble, config KubernetesConfig) error {
 	return nil
 }
 
+func UpdateCluster(d drivers.Driver) error {
+	localkube, err := Asset("out/localkube")
```

Comment:
Maybe move this to some constant? So we have only one place to update if we change the path in the future